### DISCUSSION TITLE
Using ErrorMessage with Controller

### DISCUSSION THREAD

**Discussion by kevinkli916** [react-hook-form/react-hook-form#1730] [General] [answered]
Hello, I'm new to RHF and can't seem to display an error message when using the Controller component.

Here is the code snippet I am having trouble with:

`<section>`
`    <label>React Select</label>`
    `<Controller`
        `as={Select}`
        `options={options}`
        `name="SelectTest"`
        `isClearable`
        `control={control}`
        `rules={register({ required: true })}`
    `/>`
`</section>`

`<ErrorMessage`
      `errors={errors}`
      `name="SelectTest"`
      `message={'This selection is required'}`
      `as={'p'}`
`/>`

I have also tried `{errors.SelectTest && <p>This is required</p>}` to no avail.

Any help would be appreciated.

**Comment by bluebill1049** [comment, marked as answer]
can you see `errors` object? can you reproduce the problem in a codesandbox for us to take a look?

**Reply by kevinkli916** [reply]
I do not see an `errors` object.
Here is a codesandbox [link](https://codesandbox.io/s/react-hook-form-errormessage-gm9d6?file=/src/index.js)

**Reply by bluebill1049** [reply]
`rules={{ required: true }}` check your usage.

**Reply by kevinkli916** [reply]
Thank you very much and I appreciate the quick response.
